src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadoc2.java

```diff
@@ -0,0 +1,20 @@
+/*
+SummaryJavadoc
+violateExecutionOnNonTightHtml = true
+forbiddenSummaryFragments = ^Returns the customer ID. This method returns. $
+period = .
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+public class InputSummaryJavadoc2 {
+
+    // violation 2 lines below 'Forbidden summary fragment'
```

Comment:
why two violations?

you have in test method:
```
final String[] expected = {
            "15: " + getCheckMessage(MSG_SUMMARY_JAVADOC),
        };
```

Reply:
There are two lines that are matching with the forbiddenSummaryFragments

Reply:
but in array we have only one violation.
and in inlined comment you mentioned that there are two.

if I am blind ..... ok
please share CLI output for this Input.
here is hoow to build jar of your changes - https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations#how-to-generate-all-binaries-and--alljar--too

Reply:
index.java 
```
public class index {
    /**
     * {@summary Returns the customer ID.
     *  This method returns. }
     */
    public void foo() {
    }
}
```
config.xml
```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC 
 "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
 "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="SummaryJavadoc">
            <property name="violateExecutionOnNonTightHtml" value="true" />
            <property name="forbiddenSummaryFragments"
                value="^Returns the customer ID. This method returns. $" />
                            <!-- attention to extra space before $  ^^^ -->
        </module>

    </module>
</module>
```
Output: 
```
aaayush@aaayush-mac GFG % java -jar checkstyle-10.12.5-SNAPSHOT-all.jar -c config.xml index.java
Starting audit...
[ERROR] /Users/aaayush/Desktop/GFG/index.java:3: Forbidden summary fragment. [SummaryJavadoc]
Audit done.
Checkstyle ends with 1 errors
```
@romani This is the status of the new build jar

Reply:
My bad , I blindly read it as 2 violations, but it is violation 2 lines below